src/etc/inc/filter.inc

```diff
@@ -716,6 +716,16 @@ function filter_generate_aliases()
                 $tmp_ports = implode(" ", filter_core_get_port_alias($aliased['name'], array(), $aliasObject));
                 $aliases .= "{$aliased['name']} = \"{ {$tmp_ports} }\"\n";
                 break;
+            case "dynipv6host":
```

Comment:
for addresses we try to prevent adding hooks in filter.inc, since all (non port) alias types are being handled by `update_tables.py` so there's no hard coupling between aliases and rules.